gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/Oplog.java

```diff
@@ -3427,6 +3436,9 @@ else if (tag == null) {
             CompactionRecord cr = new CompactionRecord(keyBytes, crOffset);
             getRecoveryMap().put(oplogKeyId, cr);
             drs.getDiskRegionView().incRecoveredEntryCount();
+            if (EntryBits.isAnyInvalid(userBits) || EntryBits.isTombstone(userBits)) {
+              drs.getDiskRegionView().incInvalidOrTombstoneEntryCount();
+            }
```

Comment:
I can see the calls to incRecoveredEntryCount in a couple of more places in Oplog class. Please check if the corresponding calls to incInvalidOrTombstoneEntryCount are really not required.